src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -48,13 +44,22 @@ public CommandResult execute() {
      */
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
+        final Set<String> lowercaseKeywords = convertLowercase(keywords);
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
             final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
-            if (!Collections.disjoint(wordsInName, keywords)) {
+            if (!Collections.disjoint(wordsInName, lowercaseKeywords)) {
```

Comment:
I see that you convert the name into lowercase in the Name class, but it's not clear from this chunk of code..